.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+Closes #???
+
+The following tasks have been completed:
+
+- [ ] Modified Web platform tests ([link](https://github.com/web-platform-tests/wpt/))
```

Comment:
What happens to these checkboxes in informative or editorial PR's?

Reply:
You don't check them.